packages/cli/commands/config.js

```diff
@@ -0,0 +1,23 @@
+const { addConfigOptions, addAccountOptions } = require('../lib/commonOpts');
+const list = require('./config/list');
+const setDefault = require('./config/default');
+const rename = require('./config/rename');
+
+exports.command = 'config';
+exports.describe = 'Commands for working with the config file';
+
+exports.builder = yargs => {
+  addConfigOptions(yargs, true);
+  addAccountOptions(yargs, true);
+
+  yargs
+    .command({
+      ...list,
+      aliases: 'ls',
+    })
```

Comment:
Instead of adding the alias here, can we add to the command in `config/list`?

Reply:
If we list aliases of subcommands in the parent command we don't have to sift through each subcommand to find every alias and avoid collisions. It also matches the current pattern for the `hs list` (`hs ls`) and `hs functions list` (`hs functions ls`).